packages/backend/src/database/seeds/development/05_generate_fake_data.ts

```diff
@@ -0,0 +1,151 @@
+import { faker } from '@faker-js/faker';
```

Comment:
do we always want this seed for devs?

Reply:
This seed only runs if `FAKE_DATA_LENGTH >= 1`, so unless we explicitly declare it this won't add anything